server/datastore/mysql/migrations/tables/20230328091816_EnsureUniformCollation.go

```diff
@@ -3,14 +3,170 @@ package tables
 import (
 	"bytes"
 	"database/sql"
+	"encoding/json"
 	"fmt"
 	"text/template"
 
 	"github.com/jmoiron/sqlx"
 )
 
 func init() {
-	MigrationClient.AddMigration(Up_20230315104937, Down_20230315104937)
+	MigrationClient.AddMigration(Up_20230328091816, Down_20230328091816)
+}
+
+func fixupSoftware(tx *sql.Tx, collation string) error {
+	rows, err := tx.Query(`
```

Comment:
Can we explicitly disable the `gosec` check for this? Ideally adding a comment like "string formatting must be used here, but input is not user-controllable".